kore/src/Kore/Step/Rule.hs

```diff
@@ -579,24 +621,45 @@ patternToAxiomPattern
     => Attribute.Axiom
     -> TermLike variable
     -> Either (Error AxiomPatternError) (QualifiedAxiomPattern variable)
-patternToAxiomPattern attributes pat
-  | isJust . getPriority . Attribute.priority $ attributes =
+patternToAxiomPattern attributes pat =
     case pat of
+        Rewrites_ _ (ApplyAlias_ alias params) rhs ->
+            case substituteInAlias alias params of
+                And_ _ requires lhs ->
+                    patternToAxiomPattern
+                        attributes
+                        (mkRewrites (mkAnd requires lhs) rhs)
+                _ ->
+                    error
```

Comment:
This looks like it will print the error message and the pattern on the same line. I think it would be nicer if we used `Pretty` here to also indent the pattern so the error is more readable like we do in other places in the code.